### ISSUE TITLE
I remember an option that can be accessed from the context menu to enable logging on video

### ISSUE THREAD

**Issue by JenieX** [mpvnet-player/mpv.net#548]
Normally when developing a script I would launch mpvnet via command line. Then I remembered now that there was an option to enable the logging on the video that is being played, except I can't find that option and I'm losing my mind thinking maybe there never was.

**Comment by stax76** [comment]
There was a history feature that was either removed or is subject to be removed, because I ported it to a user script, which can be found here:

https://github.com/stax76/mpv-scripts/blob/main/history.lua

There are alternative scripts you can find searching for the keywords **history** and **recent**.

https://github.com/stax76/awesome-mpv#playback

**Comment by JenieX** [comment]
Thank you for answering but I think you misunderstood me.

Say in my new script I have this line:
```js
mp.msg.warn('Hello world');
```

Usually to see that message (to my knowledge), I would have to run mpvnet through the command line:
```shell
mpvnet "my-video.mkv"
```

At some point when I was running an old version of mpvnet, I tried a new version, that new version had option under its context menu that activate the logging (debugging) on the video itself without the need to run mpvnet from the command line. In another word, I would be able to see the message `'Hello world'` when I toggle ON that option.

Now that I moved completely to the latest version, that option is no longer there!
